lib/rspec/sqlimit_equal.rb

```diff
@@ -0,0 +1,47 @@
+require "active_support/notifications"
+require "active_record"
+require "rspec"
+
+module RSpec
+  module SQLimitEqual
```

Comment:
Let's move the matcher code to the existing `sqlimit.rb` file, no need to extract it.